index.html

```diff
@@ -142,6 +142,11 @@
           A successful PATCH request MUST respond with a 2xx status code; the specific code in the 2xx
           range MAY vary according to the response body or request state.
         </p>
+        <p>
+          When receiving a PATCH request that would change the LDP interaction model of a resource
```

Comment:
@awoods your suggestion seems fine to me -- would you be willing to submit a PR against this PR with your suggested change? I'm not quite sure what you have in mind w/r/t subsection structure. (I'd also note that too much re-organization will result in merge conflicts with #45)